src/flask/schemas/chef_schema.py

```diff
@@ -0,0 +1,24 @@
+# schemas fro serializing entities
+from datetime import datetime
+
+from marshmallow import Schema, fields, post_load
+
+from src.constants.audit import Status, InternalUsers
+from src.lib.entities.sqlalchemy_orm_mapping import Chef
+
+
+class ChefSchema(Schema):
+
+    user_id = fields.Int()
+    skill = fields.Int()
+
+    @post_load
+    def create_sqlalchemy_chef_entity(self, data, **kwargs):
+        chef = Chef()
+        chef.skill = data["skill"]
+        chef.user_id = data["user_id"]
+        chef.entity_status = Status.ACTIVE.value
```

Comment:
You don't need this, your service should this for you. Also use `data.get()` instead. It is possible that a key may not exist. You can create a generic utility that makes this conversion for you by getting all the keys from and set all the attributes of the class having the same props

Reply:
done. I used data.get() instead.